.gitignore

```diff
@@ -64,3 +64,6 @@ typings/
 
 karma.conf.js
 protractor.conf.js
+
```

Comment:
Hey, @NOcode-developer, please revert the changes of this file, to remove it from this pull-request.
 In this task, you should only add your Readme, not change the settings of whole repo.

Reply:
I made changes to the .gitignore file.
Please check if everything is done correctly.